plugins/vault/main.py

```diff
@@ -0,0 +1 @@
+../../core/plugins/utils/python_plugin_runner.py
```

Comment:
This doesn't look right :smile:

Reply:
yep all good:

$ readlink plugins/*/main.py
../../core/plugins/utils/python_plugin_runner.py
../../core/plugins/utils/python_plugin_runner.py
../../core/plugins/utils/python_plugin_runner.py
../../core/plugins/utils/python_plugin_runner.py
../../core/plugins/utils/python_plugin_runner.py
../../core/plugins/utils/python_plugin_runner.py
../../core/plugins/utils/python_plugin_runner.py
../../core/plugins/utils/python_plugin_runner.py
../../core/plugins/utils/python_plugin_runner.py
../../core/plugins/utils/python_plugin_runner.py

Reply:
Perfect. Thanks!